modules/apps/object/object-test/src/testFunctional/tests/Picklist.testcase

```diff
@@ -658,10 +658,9 @@ definition {
 		SelectFrame(locator1 = "Picklist#IFRAME_PICKLIST_ITEM_MODAL");
 
 		AssertValue(
-			key_itemKey = "PicklistItemKeyTest",
 			key_itemName = "Picklist Item Name Test",
-			locator1 = "Picklist#ADD_PICKLIST_ITEM_NAME",
-			value1 = "PicklistItemKeyTest");
+			locator1 = "Picklist#VIEW_PICKLIST_ITEM_NAME",
```

Comment:
You can use the same path from the previous comment (if it works), and then make the assertion of the value based on the Label, more assertive, and not in the value itself.